.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,14 +4,9 @@ about: Suggest an idea for this project
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**MSAL is the preferred auth library for Microsoft**
```

Comment:
idk about this statement..maybe something like "MSAL is the recommended/preferred auth library for use with the Microsoft identity platform"...not sure.

Reply:
I like the suggestion

---
In reply to: [299109879](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1623#discussion_r299109879) [](ancestors = 299109879)